### PR TITLE
Create and remove books

### PR DESCRIPTION
**Pull request by Metsanpeitto**
:ok_man:  In this step, I configured the Redux Store and wrote actions and reducer for adding and removing books. I also added some UI elements to the components and make them dispatch actions.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Metsanpeitto 👋,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 - Make simple calculations.
 - Read a random math-related quote.
 
+
```
Comment (mikenath223) [inline README.md:9]: - [ ] In `src/redux/books/books.js` reducer kindly ensure there are only two cases: `ADD_BOOK` and `REMOVE_BOOK`. Please remove the `FILTER_BOOKS` case. See project requirement [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_create_and_remove_book.md#:~:text=Write%20your%20reducer%20and%20export%20it%20as%20default%3A).

Comment (mikenath223) [reply README.md:9]: - [ ] OPTIONAL - Also, in the `REMOVE_BOOK` case you can simply use the Array.prototype.filter method to filter out the book containing the payload id.

Comment (Metsanpeitto) [reply README.md:9]: :ok_hand: Thanks for pointing it out. 
- :books:  I have already removed the FILTER_BOOKS case from the books reducer and created a dedicated reducer for it.
- :+1: I have also modified the REMOVE_BOOK to filter items out.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**